### ISSUE TITLE
Increase max width of rubric points text box

### ISSUE THREAD

**Issue by joelostblom** [PrairieLearn/PrairieLearn#10850]
It is currently not possible to see the points associated with each rubric item when using negative scoring and partial points:

![Image](https://github.com/user-attachments/assets/d6da9493-2f16-4ed5-8bf9-b5df91266436)

I tried making the window full screen on an ultrawide monitor but it seems like the text box is fixed to a too narrow max width.

Edit: This was on Firefox (on Ubuntu)

**Comment by nwalters512** [comment]
I can reproduce this on macOS Safari, which always shows the up/down stepper buttons.

Relevant code:

https://github.com/PrairieLearn/PrairieLearn/blob/38b83c3a1e5cb95451863ffaa6a531826eae745d/apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts#L311

**Comment by nwalters512** [comment]
@Victorsss-Orz heads up that I think your recent work here will fix this once the instance question rubric editor is updated to use the new component. Can you confirm if that's true?

**Comment by Victorsss-Orz** [comment]
Yeah the Preact component has `width: 5rem` which is wide enough for this. This also looks to be already fixed in this Modal as I'm seeing `width: 5rem` as well.

**Comment by nwalters512** [comment]
Looks like @miguelaenlle fixed this in #12048 🎉
